config/wdio/conf.js

```diff
@@ -1,10 +1,10 @@
 const localIP = require('ip');
```

Comment:
If we are moving these to configs, would it makes sense to match the config name you'd expect? `wdio.conf`?